src/lib/modules/coverage/contract_source.js

```diff
@@ -294,13 +267,21 @@ class ContractSource {
 
     } while(nodesRequiringVisiting.length > 0);
 
+    this._generateCodeCoverageForBytecode(trace, coverage, sourceMapToNodeType, this.contractBytecode);
+    this._generateCodeCoverageForBytecode(trace, coverage, sourceMapToNodeType, this.contractDeployedBytecode);
+
+    return coverage;
+  }
+
+  _generateCodeCoverageForBytecode(trace, coverage, sourceMapToNodeType, contractBytecode) {
     var contractMatches = true;
-    for(var contractName in this.contractBytecode) {
-      var bytecode = this.contractBytecode[contractName];
+    for(var contractName in contractBytecode) {
```

Comment:
I think we're better off using `for...of` in here (and replace `var` with `let/const` while we're at it)

Reply:
contractBytecode is not iterable
Only `for...in` will work on it

Reply:
Ha. If this was typed, I'd know! :D